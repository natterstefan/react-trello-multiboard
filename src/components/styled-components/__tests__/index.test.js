import React from 'react'
import { shallow } from 'enzyme'
// https://github.com/styled-components/jest-styled-components
import 'jest-styled-components'

import { BlockContainer, ScrollContainer } from '../index'

describe('StyledComponents', () => {
  test('should render an BlockContainer with the proper styling', () => {
    expect(shallow(<BlockContainer />)).toMatchSnapshot()
  })

  test('should render an ScrollContainer with the proper styling', () => {
    expect(shallow(<ScrollContainer />)).toMatchSnapshot()
  })
})
