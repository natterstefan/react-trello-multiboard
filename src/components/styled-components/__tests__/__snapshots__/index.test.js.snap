// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledComponents should render an BlockContainer with the proper styling 1`] = `
.c0 {
  display: block;
  padding: 10px 0;
}

<div
  className="c0"
/>
`;

exports[`StyledComponents should render an ScrollContainer with the proper styling 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  max-width: 100vw;
  min-width: 0;
  overflow-x: auto;
  padding: 10px 0;
}

.c0:after {
  content: '';
  -webkit-flex: 0 0 40px;
  -ms-flex: 0 0 40px;
  flex: 0 0 40px;
}

<div
  className="c0"
/>
`;
